packages/backend/code/index.ts

```diff
@@ -53,6 +53,29 @@ const endTurn = (roomCode: string): void => {
   updateGameForRoom(rooms[roomCode]);
 };
 
+/**
+ * Randomize teams in a room.
+ * @param room
+ */
+const randomizeTeams = (room: Room): void => {
+  let players: PlayerData[] = room.players;
+  players = shuffle(players);
+
+  // Determine initial team color
+  let evenTeam: Team = Color.Red;
+  let oddTeam: Team = Color.Blue;
+
+  if (Math.random() < 0.5) {
+    oddTeam = Color.Red;
+    evenTeam = Color.Blue;
+  }
+
+  // Assign new teams to players
+  for (let i = 0; i < players.length; i++) {
+    players[i].team = i % 2 === 0 ? evenTeam : oddTeam;
+  }
+};
+
```

Comment:
Also need `updateGameForRoom(room)` here to refresh the player lists for the client.

```suggestion
  for (let i = 0; i < room.players.length; i++) {
    room.players[i].team = i % 2 === 0 ? evenTeam : oddTeam;
  }

  updateGameForRoom(room);
```